consts/consts.go

```diff
@@ -0,0 +1,5 @@
+package consts
```

Comment:
do we really need a new package for consts :thinking: maybe we can use `cmd` package for this purpose and create that const/value there :thinking:

Reply:
I'm not keen on having an import loop in the packages... However, this isn't an issue I suppose

Reply:
done

Reply:
Import loops are not allowed in Go